.evergreen.yaml

```diff
@@ -88,7 +89,12 @@ functions:
         working_dir: src/atlas-service-broker
         script: |
           docker login --username ${docker_username} --password ${docker_password} ${docker_repo}
+
           docker push ${docker_repo}/${docker_name}:${version}
+          if [[ ${version} =~ [0-9]+.[0-9]+$ ]]; then
```

Comment:
This should include the third stanza of the version number (i.e. the patch version).

Reply:
cool, np.